specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2021-02-10/service.json

```diff
@@ -19013,6 +22207,25 @@
       },
       "discriminator": "instanceType"
     },
+    "TenantDB.InputEndpoint": {
```

Comment:
remove this

Reply:
removed

---
In reply to: [587900918](https://github.com/AsrOneSdk/azure-rest-api-specs/pull/16#discussion_r587900918) [](ancestors = 587900918)

Reply:
@nilgupta6 this is the InputEndpoint regarding which you asked in the other comment. Seems like its coming with a namespace.

---
In reply to: [591476529](https://github.com/AsrOneSdk/azure-rest-api-specs/pull/16#discussion_r591476529) [](ancestors = 591476529,587900918)